### ISSUE TITLE
packages.config support for nuget

### ISSUE THREAD

**Issue by Nysosis** [versioneye/versioneye#628]
It would be useful if you could parse the packages.config file in a .net project to search for dependencies too. None of our projects are on .net core, so we're not using project.json, and we don't specify our dependencies in the nuspec file, as we let nuget take care of updating the metadata when packaging the project (in fact no project I've ever worked on has ever explicitly specify dependencies in the .nuspec file - though I'm sure there are many that do).

**Comment by reiz** [comment]
Hi @Nysosis. Many Thanks for your feedback. Somebody else also mentioned this point. I guess @timgluz is already working on this.

**Comment by Nysosis** [comment]
Ah thanks, I didn't quite spot that this is the repo for the website rather than the actual tool, I just followed the issues link from the blog post.

Found the [pull request](https://github.com/versioneye/versioneye-core/pull/35), so I'll wait on that.

**Comment by reiz** [comment]
@Nysosis I just merged the pull request from @timgluz and deployed it already to production. The unit tests for the new parser are passing, but please test it with your own files and confirm that it works for you as expected. If it doesn't work like expected open another ticket for that issue :)
